pkgs/by-name/vc/vcpkg/package.nix

```
@@ -65,7 +65,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     '';
 
   meta = {
-    description = "C++ Library Manager";
+    description = "C++ Library Manager for Windows, Linux, and MacOS";
```

The official capitalization is “macOS” ([source](https://apple.com/macos)).

```suggestion
    description = "C++ Library Manager for Windows, Linux, and macOS";
```

---

I simply copy pasted the description they put on their GitHub repo.

---

OK. [I just submitted an upstream issue for this problem.](https://github.com/microsoft/vcpkg/issues/40433)